site/src/analytics-os-boards.ts

```diff
@@ -0,0 +1,104 @@
+import "@google-web-components/google-chart";
+import { GoogleChart } from "@google-web-components/google-chart";
+import {
+  css,
+  customElement,
+  html,
+  LitElement,
+  property,
+  PropertyValues,
+  query,
+} from "lit-element";
+import { AnalyticsDataCurrent } from "../../worker/src/data";
+
+@customElement("analytics-os-boards")
+export class AnalyticsOsBoards extends LitElement {
+  @property({ attribute: false }) public currentData?: AnalyticsDataCurrent;
+
+  @property({ type: Boolean }) public isMobile = false;
+
+  @property({ type: Boolean }) public isDarkMode = false;
+
+  @query("google-chart") private _chart?: GoogleChart;
+
+  protected firstUpdated(_changedProperties: PropertyValues) {
+    super.firstUpdated(_changedProperties);
+    window.addEventListener("resize", () => {
+      this._chart?.redraw();
+    });
+  }
+
+  render() {
+    if (
+      this.currentData === undefined ||
+      this.currentData.operating_system.boards
+    ) {
+      return html``;
+    }
+
+    const sortedBoards = Object.keys(
+      this.currentData.operating_system.boards
+    ).sort(
+      (a, b) =>
+        this.currentData!.operating_system.boards[b] -
+        this.currentData!.operating_system.boards[a]
+    );
+
+    const rows = sortedBoards.map((board) => [
+      board,
```

Comment:
Maybe give this a nice name with a mapping?
```suggestion
      boardName[board] || board,
```
```ts
const boardName = {
  "rpi3": "Raspberry Pi 3",
  "rpi4": "Raspberry Pi 4",
  ...
}
```